tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -891,9 +911,13 @@ _prevctx = localctx
 >>
 
 
-LexerFile(lexerFile, lexer, namedActions) ::= <<
-<fileHeader(lexerFile.grammarFileName, lexerFile.ANTLRVersion)>
-package parser
+LexerFile(file, lexer, namedActions) ::= <<
+<fileHeader(file.grammarFileName, file.ANTLRVersion)>
+<if(file.genPackage)>
+package <file.genPackage><! // <file.grammarName>!><! why can't we access file.grammarName here? !>
```

Comment:
Right, but the other file templates have an import comment, and file.grammarName works in them. This comment is saying file.grammarName doesn't work here, so I understand why it's left out, but it's a weird exception. Any idea why this is the case?

It might be clearer to change the template comment wording from a question to a statement like "file.grammarName does not work here [because...]".